lib/ConsoleAgent.js

```diff
@@ -135,18 +135,22 @@ ConsoleAgent.prototype.injection = function(require, debug, options) {
   }
 
   function getCallerFuncLocation() {
-    var location = {};
-    try { throw new Error(); }
-    catch (e) {
-      //Magic 4 is:                                                                       4
-      //[Error string]->[getCallerFuncLocation]->[ConsoleMessage]->[Wrapped function]->[Caller]
-      var callerStackLine = e.stack.split('\n')[4];
-      var _location = /\((.*):(\d+):(\d+)\)$/.exec(callerStackLine);
-      location.url = _location[1];
-      location.line = _location[2];
-      location.column = _location[3];
-    }
-    return location;
+    var oldPrepareStackTrace = Error.prepareStackTrace,
+        callerFrame;
+
+    Error.prepareStackTrace = function(error, stack) { return stack; }
```

Comment:
I'd prefer to not override builtin methods, but as long as this works, I can live with it.